apps/src/netsim/NetSimLobby.js

```diff
@@ -60,14 +63,28 @@ var NetSimLobby = function (connection) {
    * @private
    */
   this.connection_ = connection;
-  this.connection_.statusChanges.register(this, this.refreshLobby_);
+  this.connection_.statusChanges.register(this, this.refresh_);
```

Comment:
The ObservableEvent being used here binds the first argument to `register` as this for the second argument.  If you'd like, we can change it to take one argument and bind the methods explicitly - it would probably be more idiomatic.